RELEASE/scripts/autoscend.ash

```diff
@@ -5,6 +5,7 @@ since r20249; // Process text generated by hewn rune spoon moon dune whatever no
 
 	Accessory scripts can import autoscend.ash
 
+	If updating since revision, update .github/workflows/ci.yml and cached kolmafia.jar
```

Comment:
If you're updating the cached jar I suppose you don't need to update the ENVs too! But seems right to me